app/src/main/java/uk/co/neuralcubes/neuralates/controller/RobotController.java

```diff
@@ -32,7 +34,16 @@ public void unlink(){
     @Subscribe
     public synchronized void updateAcelerometer(MuseHandler.AccelerometerReading reading){
```

Comment:
They might be accessed in parallel yep.

On Tue, 19 Apr 2016 at 15:07 Ramón Argüello <notifications@github.com>
wrote:

> In
> app/src/main/java/uk/co/neuralcubes/neuralates/controller/RobotController.java
> <https://github.com/neuralcubes/neuralates/pull/15#discussion_r60237028>:
>
> > @@ -32,7 +34,16 @@ public void unlink(){
> >      @Subscribe
> >      public synchronized void updateAcelerometer(MuseHandler.AccelerometerReading reading){
>
> Even if they're asynchronous, 'synchronized' wouldn't be needed if they
> always come from the same thread. Happy to the leave them in if we're not
> sure though.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/neuralcubes/neuralates/pull/15/files/8494d8e77337c004d3acb7745e61bf4a28b61da4#r60237028>
>